src/project-conf/index.ts

```diff
@@ -74,13 +85,14 @@ export async function getProjectConfigurationElements(workspaceFolder: Uri) {
           compileArgs: buildConfig?.compileArgs,
           ninjaArgs: buildConfig?.ninjaArgs,
           buildDirectoryPath: buildDirPath,
-          sdkconfigDefaults: buildConfig?.sdkconfigDefaults?.map(
-            (path) => Uri.joinPath(workspaceFolder, path).fsPath
-          ),
-          sdkconfigFilePath: buildConfig?.sdkconfigFilePath
-            ? Uri.joinPath(workspaceFolder, buildConfig.sdkconfigFilePath)
-                .fsPath
-            : buildConfig?.sdkconfigFilePath,
+          sdkconfigDefaults: (await pathExists(buildConfig?.sdkconfigDefaults))
```

Comment:
Need to consider that sdkconfigDefaults is an array of files so this part of code would fail.